GuitarStore.cpp

```diff
@@ -0,0 +1,38 @@
+#include <iostream>
+
+#include "Guitar.h"
+#include "Airliner.h"
+
+#include "GuitarStore.h"
+
+GuitarStore::GuitarStore() { std::cout << "Constructing Guitar Store " << std::endl; }
+    
+GuitarStore::~GuitarStore() { std::cout << "Destroying GuitarStore" << std::endl; }
+
+void GuitarStore::info()
+{
+    std::cout << "Store Guitar info:\n Model:" << this->guitar.model << "\n String number: "<< this->guitar.stringNum << "\n Fret number: "<< this->guitar.fretNum << "\nModified to: " << std::endl;
+}
+
+void GuitarStore::modifyGuitar(double gauge, char model, int frets, int stringNum)
+{
+    std::cout << "Modifying guitar:\n Model:" << guitar.model << "\n String number: "<< guitar.stringNum << "\n Fret number: "<< guitar.fretNum << "\nModified to: " << std::endl;
+
+    strings.gauge = gauge;
+    guitar.model = model;
+    guitar.fretNum = frets;
+    guitar.stringNum = stringNum;
+    
+    guitar.restringGuitar(strings);
+}
+
+void GuitarStore::loadPlane(int guitarNum, int passengers)
+{
+    plane.beginBoarding(passengers);
+    while( (plane.capacity && guitarNum) > 0 )
```

Comment:
What you wrote here is evaluated as the following:
```
auto expr = plane.capacity && guitarNum;
while( expr > 0 )
```
when you write `plane.capacity && guitarNum`, the CPU checks if either of those values is != 0.
essentially you get: `(is not 0) && (is not 0)`.   if either of them is zero, then `expr` is `false`, otherwise `expr` is `true`.
then it compares if `expr` is `> 0`.  

This is obviously not what you want. 
What you want is to compare if `plane.capacity` is greater than 0, and if `guitarNum` is also greater than 0.
You need to be explicit when you write logic that compares values.
- [ ] revise